src/Components/SingleArticle.jsx

```diff
@@ -1,29 +1,41 @@
 import React, { useEffect, useState } from "react";
 import { fetchData } from "../utils/api";
 import { useParams } from "react-router-dom";
+import CommentsList from "./CommentsList";
 
-function ArticleCard () {
+function SingleArticleList () {
     const [article, setArticle] = useState({})
+    const [isLoading, setIsLoading] = useState(true)
     const { article_id } = useParams()
 
     useEffect(() => {
+        setIsLoading(true)
         fetchData(`/articles/${article_id}`).then(({article}) => {
```

Comment:
It would be better to avoid passing the endpoint as an argument and deal with it in api.js instead to make it tidy and less confusing